test/testutil/outcome.hpp

```diff
@@ -148,7 +148,7 @@
     }                                                                     \
   }
 
-#define EXPECT_OUTCOME_SUCCESS(_result_, _expression_)                  \
+#define EXPECT_OUTCOME_SUCCESS(_expression_)                            \
```

Comment:
Why? It can be needed to get and serve result sometimes.
I guess you should use ASSERT_OUTCOME_SUCCESS_TRY in your case.